BackofficeBundle/Resources/views/BackOffice/Include/NavigationPanel/Menu/Administration/api_client.html.twig

```diff
@@ -3,10 +3,7 @@
         id="nav-api_client"
         data-url="{{ path('open_orchestra_api_api_client_list') }}"
         data-add="{{ path('open_orchestra_backoffice_api_client_new') }}"
```

Comment:
not sure.I prefer keeping only datatable configuration in this PR. @itkg-ngilain